examples/generate_all_certs

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+Dash="\n\e[38;2;223;223;16m────────────────────────────────────────────────────"
+Reset="\e[0m"
+
+# change to script folder
+SCRIPT=$(realpath -s "$0")
+CERT_HOME=$(dirname "$SCRIPT")
+
+#——————————————————————————————— process ssl files in examples/puma
+echo -e "$Dash process ssl files in examples/puma$Reset"
+cd $CERT_HOME/puma
+ruby ../generate_server_test.rb
+printf "Done with ../generate_server_test.rb\n\n"
+
+if [ -f server.p12 ]; then rm server.p12; fi
```

Comment:
`rm -f <file_name>` does not throw an error if the file does not exist.

Reply:
I don't know, I thought the code should work if one cleared/removed all the certs, hence no error...

Reply:
It's a suggestion to use more simple and concise code :)

Reply:
Thanks, I missed that.  Still a bit distracted…

Reply:
Fixed...